docs/user-guide.md

```diff
@@ -5,7 +5,7 @@ To best understand the benefits of Akri and jump into using it, we recommend you
 To get started using Akri, you must first decide what you want to discover and whether Akri current supports a protocol that can be used to discover resources of that type. To see the list of currently supported protocols, see our [roadmap](./roadmap.md).
 
 ### Setting up your cluster
```

Comment:
How does a user get to this page? Should we add a link onto the front page README?

Reply:
Seems like a good thing to add under the [Documentation](https://github.com/deislabs/akri#documentation) section on the base README. Can do

Reply:
Aah, sorry, github search did not reveal that. I think the name is fine. But agree with Brian, we should also have it somewhere higher up.